fvtr/gccgo/gccgo.exp

```diff
@@ -38,6 +38,16 @@ $at_dir/bin/$env(AT_TARGET)-gccgo is missing."
 	set rc 1
 }
 
+
+printit "Test if all 32-bit go libraries were removed..."
```

Comment:
@tuliom Comment says 32-bit libraries, I could not see the check for 32 bit. Not sure if that is needed.

Reply:
@RajalakshmiS It's checking under 'lib/', which is where 32-bit libraries are installed.
Both ppc64 and ppc64le install 64-bit libraries under 'lib64/'.

Reply:
ok.